### ISSUE TITLE
Demos getting overwritted - No matchid?

### ISSUE THREAD

**Issue by dreaNN1337** [splewis/get5#762]
If making a feature request, you should delete all the pre-filled text here.
If reporting a bug, fill in the following sections:

### Expected behavior
When im startig more matches on the same map i want demos from every match.

### Actual behavior
The demo will be created but its getting overwritted, because it has no Matchid.
See screenshot. [https://s20.directupload.net/images/220618/p3unqdms.jpg](url)

### Steps to reproduce
1) Starting a Scrim or Match
2) End the Match
3) Look on into your demos folder (if you have created one)
4) You will see one demo
5) Repeat step 1-3
6) No new demo - the old demo is overwritted

- Plugin version: 0.7.2
- sourcemod_version = 1.10.0.6542

**Comment by PhlexPlexico** [comment]
I believe you should be able to change the `get5_demo_name_format` to include a timestamp to avoid this

**Comment by dreaNN1337** [comment]
> I believe you should be able to change the `get5_demo_name_format` to include a timestamp to avoid this

Good idea but TIME dont work because it creates only the date and _Hour
So a new demo will overwrite the old again.

**Comment by dreaNN1337** [comment]
> I believe you should be able to change the `get5_demo_name_format` to include a timestamp to avoid this

That works if i set _M for minutes at the end too.
Thanks for your help!

**Comment by PhlexPlexico** [comment]
Going to close this since it was resolved.

Thanks!
